### DISCUSSION TITLE
How to put stamp text on each page

### DISCUSSION THREAD

**Discussion by Mallinath-debug** [MatthiasValvekens/pyHanko#167] [Q&A] [answered]
How to put stamp text on each or every page

**Comment by MatthiasValvekens** [comment, marked as answer]
Hi @Mallinath-debug,


Sorry, I don't understand your question. If you just want to stamp every page, you should be able to simply call the `.apply(...)` method on any `BaseStamp` instance in a loop, incrementing the `dest_page` every time.

Unless you're asking how to create a _signature_ that appears with a stamp on every page... because that's impossible without violating the PDF standard. There are workarounds, but they're not pretty.

Anyway, you'll need to provide more details than that :).

**Reply by Mallinath-debug** [reply]
I  just want stamp on every page , I will try your solution i.e.call the .apply(...) method on BaseStamp instance in a loop
